### ISSUE TITLE
WL Pet dmg seems to be nerfed 30%+

### ISSUE THREAD

**Issue by hollodrim** [WarEmu/WarBugs#13270]
I don't know what just happened, but WL pet dmg has been nerfed substantially. auto hits, brutal pounce, coordinated strike, and pet skills from trained to kill (didn't test other 2 buffs) dmg has been nerfed by 20%-30% from what it was from the FIRST patch this week, and BEFORE that patch. And I don't see any notes of a hotfix so I assume this is a bug or a ninja nerf. Can a GM/Admin care to elaborate. A LOT harder to kill anything now even clothies...

Please tell me its a bug. The first patch this week already nerfed our bread and butter coordinated strike by an absurd amount (30%) 

EDIT: Just tested dmg for nornal non pet skills and cleave limb (-150 dmg) / Forced Opportunity (-100ish) / Pack assault (-100ish), Lio's fury (-150), Sundering chop (-100)

Is this intended?

**Comment by sioding** [comment]
If you have testied thing you - can you please add your screenshots?

**Comment by hollodrim** [comment]
I can take screenshots if you want or are you guys already testing it? The servers are going down again for a pet hotfix so this data is worthless now isn't it?... Can you guys please write down these hotfix changes so we know what's up instead of getting upset when you guys ninja nerf dmg by 20%-30%?

I know you guys are busy, but this class balancing weekly is too much. We get used to our classes then it gets nerfed. WL got nerfed so much already after Azzazel's changes and the revert. Torq's changes this week was horrific. Nerfing coordinate strike which is our bread and butter for pet spec, and then saying if we use train to threathened, there might be a dmg boost? Who uses that pet buff over trained to kill for guardian spec? Why couldn't Torq just adjust the dmg for coordinated stike instead of nerfing it? WLs were fine before this weekend's patch and before this ninja nerf.

I'm sorry for going on a rant, I know you guys mean well. This is just a huge ninja nerf. Do you want me to test after this hotfix? or are you guys already testing it?

**Comment by sioding** [comment]
I don´t wanna sound rude - but tbh - we need good proofs for the dev who is pulling this ticket. As a side note - bugreports are not forum posts. Only facts are fine in here. Your personal opinion or your constructive critisicm is welcome on our forum. So please add some testting screens with a good readable combatlog here. Thanks in advance.

**Comment by hollodrim** [comment]
Okay I can do that now. But the servers has been down for another pet hotfix. Will the dmg nerf be the same or did you guys change it again? That's what I asked above, because if you nerfed the dmg again, the data will be different

**Comment by Torquemadra** [comment]
Its really simple, you see the tooltip values? Thats what you should be getting, not what you were getting with full axeman/hunter and trained to kill co ordinated strike which synergized pet strength incorrectly and led to numbers far and away from what they should have been.

Baring any critical hits, baring any Melee/heal/magic/ranged power stats or +damage ability/tactics, your abilities, pets included, will be doing exactly what the tooltip says within a tolerance of a couple of points of damage while we tweak whats being rounded up here and there.

There is no "damage nerf", there has only been corrections, now abilities for the very most part will adhere to the numbers that are on tooltips and obey client logic accordingly. The "first patch" yesterday erroneously allowed pets the use of such tactics as flanking and the use of such abilities as Primal Frenzy, thats fixed.

If you are claming "ninja nerfs" or "bugs" then all you have to do is get a screenshot of abilities doing LESS than the tooltip by a factor of more than a couple of points and just so you know, I was just on live and verified the damage formulas with the White Lion and the pet.

**Comment by SecretsOTheP** [comment]
You're also running Trained to Kill with what I assume is no added points in Guardian - that is not how the class is meant to work. You'll need to run Trained to Threaten/Trained to Hunt if running Axeman/Hunter cookie cutter. (see: build attached)

I run this and use trained to hunt @ 70. It works better than Trained to Threaten for my playstyle because of the damage over time from Trained to Hunt + the extra points in Path of the Hunter, which also boosts the base damage of Coordinated Strike. The pet abilities become extremely strong in Hunt compared to Threaten, and the pet autoattacks are boosted by half of your strength.

http://waronlinebuilder.org/#career=wl;11;13;0;5:9:;5:9:13:;;;0:0:0:0:


In comparison to other patches' damage amounts...

-Coordinated Strike's secondary hits from the pet should hit about 100 base damage less. I was running two extra points in Axeman before, I have since moved them to hunter to make up for this.

-Autoattack for the pet and player remains unchanged from 2 weeks ago.

-Pet abilities are considerably higher, but lower if you are running a stance that you don't have mastery points in.

* With the best set in the game that is easily obtainable (4pc beastlord, 5pc vanquisher (no helm), 2pc genesis, random stat jewelry) my effective strength/crit on the pet in all stances is actually HIGHER than before. 

* With that setup, I have 30% crit in TTT and 1050 strength. Prior to the patch, my pet had NO critical chance bonus and about 881 strength, as the pet only got your strength and weaponskill from items in trained to kill. 

-Guardian gets a bigger boost to pet damage than other specs; as mastery points in Guardian increase the damage while in Trained to Kill, and the stance itself boosts the weaponskill and strength of the pet resulting in higher autoattacks from the pet itself. Leonine Frenzy also had its internal cooldown reduced to 500ms, that was omitted from the patch notes and the tooltip will be updated soon.

-Throat Bite no longer deals damage as a result of the hotfix. It was overperforming when combined with Leonine Frenzy and Brutal Pounce, you could pratically one shot a healer.

So to recap:

Guardian is a lot more viable now if you want pet damage, but you lose the heal debuff or pounce.
Axeman/Hunter is still very viable and you shouldn't run Trained to Kill while using it.
One player's ability damage values changed (Coordinated Strike).
The changes were made to align tooltips, not to nerf WL. If anything, they've gotten a buff, but you need to examine the patch and parse a lot.

**Comment by sioding** [comment]
Thanks to @SecretsOTheP for given that nice answer. Close for now.
